jupyterhub_moss/form/option_form.css

```diff
@@ -57,7 +57,13 @@
 
 #environment_simple {
   text-overflow: ellipsis;
-  max-width: 20em;
+  max-width: 100%;
+  font-size: 1.2em;
+  padding: 1.5ex 1em 1ex 1em;
+}
+
+#runtime_simple {
+  padding: 1.0ex 0.5em 0.5ex 0.5em;
```

Comment:
Added the `grid-gap` in https://github.com/silx-kit/jupyterhub_moss/pull/99/commits/d15b6f7bf1efcd35ae8030b4c480446287ce971b
I re-adjusted the padding in the _Jupyter environment_ menu to be `1rem` all around, that should fix the asymmetry issues and still make it taller than the other menu, being easier to spot. Let me know if you like it.

Reply:
To me, same input type should have same style.

Since pages are based on jinja templates, it's possible to provide a way to override the default style, I'll look at it an provide an example

Reply:
it's not a big deal for us :slightly_smiling_face:
I reverted the increased padding in the environment menu in https://github.com/silx-kit/jupyterhub_moss/pull/99/commits/070bc64cf867fe349195a250325397f3e3fa2879

Reply:
Thanks!

I made PR #101 to provide a way to override the styles of the options form.